msb/network/pubsub/factories.py

```diff
@@ -55,6 +55,7 @@ def register_subscriber(name, subscriber_class: Subscriber, config):
 
 register_subscriber("zmq", ZMQ_Subscriber, ZMQConf)
 register_subscriber("mqtt", MQTT_Subscriber, MQTTConf)
+register_subscriber("zmq_raw", ZMQ_Subscriber, ZMQConf)
```

Comment:
Why register a new subscriber under a new name but with the same parameters?

Reply:
this was the initial idea to solve the packstyle-problem - removed it